src/main/java/marquez/api/Job.java

```diff
@@ -12,20 +12,23 @@
   private final Timestamp nominalTime;
   private final String category;
   private final String description;
+  private final String location;
 
   public Job(
       final UUID guid,
       final String name,
       final String ownerName,
       final Timestamp nominalTime,
       final String category,
```

Comment:
Should we remove category?

Reply:
this is an API model, which I tried not to modify much. I think this was initially something I had to do to get a test to pass, but is no longer needed. It might make more sense to just undo my changes to the API models since it's causing some confusion.

Reply:
I reverted this unnecessary change to the Job API model, so it should now be in it's previous state.

Reply:
This should be good now -- there should not have been any API model changes in this PR since you'll be touching those for the resource layer.